lib/rollbar/rake.rb

```diff
@@ -20,6 +20,7 @@ def self.skip_patch
     end
 
     def self.patch?
+	  return false if rake_version.nil?
```

Comment:
Can you change this to `return false unless rake_version`.

Also, you have two many spaces in the indentation.

Thanks!